n_ftoa.c

```diff
@@ -287,8 +287,7 @@ fmtflt(char *str, size_t *len, size_t size, JNUMBER fvalue, int width,
         if (fpos > 0)	/* Omit trailing fractional part zeros. */
             while (omitcount < fpos && fconvert[omitcount] == '0') {
                 omitcount++;
-            }
-        else {	/* The fractional part is zero, omit it completely. */
+            } else {	/* The fractional part is zero, omit it completely. */
```

Comment:
tabs / spaces

Reply:
which ironically seems like it should have been picked up by `astyle`... :raised_eyebrow:

Reply:
Fixed and added braces to the opening `if`, which I think makes the code clearer.